Description: Set the source encoding to fix the build failures with Java 9+
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -133,6 +133,7 @@
 <target name="compile" depends="prepare,check_for_optional_packages,compile-for-java2">
     <mkdir dir="${build.classes}" />
     <javac srcdir="${src.dir}" destdir="${build.classes}"
+           encoding="ISO-8859-1"
            debug="${debug}" deprecation="${deprecation}" optimize="${optimize}">
         <classpath refid="base.classpath" />
         <exclude name="**/JTidyHTMLParser.java" unless="jtidy.present" />
@@ -257,6 +258,7 @@
              destdir="${javadoc.dir}" author="true" version="true"
              windowtitle="${Name} ${version} API" doctitle="${Name}"
              additionalparam="-notimestamp"
+             encoding="ISO-8859-1"
              footer="Copyright &amp;copy; 2000-2008 Russell Gold. See &lt;a target=&quot;_top&quot; href=&quot;../license.html&quot;&gt;license agreement&lt;/A&gt; for rights granted." >
         <classpath refid="base.classpath"/>
     </javadoc>
